examples/examples/capture-webcam-h264/capture-webcam-h264.rs

```diff
@@ -0,0 +1,280 @@
+                .long("fullhelp"),
+        )
+        .arg(
+            Arg::new("debug")
+                .long("debug")
+                .short('d')
+                .help("Prints debug log information"),
+        )
+        .arg(
+            Arg::new("device")
+                .required_unless_present("FULLHELP")
+                .takes_value(true)
+                .short('v')
+                .long("device")
+                .help("Video capture device to stream from. Eg. /dev/video0"),
+        );
+
+    let matches = app.clone().get_matches();
+
+    if matches.is_present("FULLHELP") {
+        app.print_long_help().unwrap();
+        std::process::exit(0);
+    }
+
+    let debug = matches.is_present("debug");
+    if debug {
+        env_logger::Builder::new()
+            .format(|buf, record| {
+                writeln!(
+                    buf,
+                    "{}:{} [{}] {} - {}",
+                    record.file().unwrap_or("unknown"),
+                    record.line().unwrap_or(0),
+                    record.level(),
+                    chrono::Local::now().format("%H:%M:%S.%6f"),
+                    record.args()
+                )
+            })
+            .filter(None, log::LevelFilter::Trace)
+            .init();
+    }
+
+    let device_path = matches.value_of("device");
+    let device_path = if let Some(device_path) = &device_path {
+        let device_path: PathBuf = device_path.into();
+        if !device_path.exists() {
+            return Err(Error::new(format!(
+                "video capture device: '{}' not exist",
+                device_path.display()
+            ))
+            .into());
+        }
+
+        device_path
+    } else {
+        return Err(Error::new(format!("video capture device not selected").into()).into());
+    };
+
+    let max_fps = 60;
+
+    // Create a video track
+    let video_track = Arc::new(TrackLocalStaticSample::new(
+        RTCRtpCodecCapability {
+            mime_type: MIME_TYPE_H264.to_owned(),
+            ..Default::default()
+        },
+        "video".to_owned(),
+        "webrtc-rs".to_owned(),
+    ));
+
+    println!("stream video from camera: {:?}", device_path);
+
+    let device_path_clone = device_path.clone();
+    let video_track_clone = Arc::clone(&video_track);
+
+    let pool = tokio_util::task::LocalPoolHandle::new(1);
+
+    pool.spawn_pinned(move || {
+        async move {
+            // Connect to the webcam
+            let mut device = h264_webcam_stream::get_device(&device_path_clone).unwrap();
+            let mut stream = h264_webcam_stream::stream(&mut device, max_fps).unwrap();
+
+            // It is important to use a time.Ticker instead of time.Sleep because
+            // * avoids accumulating skew, just calling time.Sleep didn't compensate for the time spent parsing the data
+            // * works around latency issues with Sleep
+            let mut ticker = tokio::time::interval(Duration::from_millis(33));
```

Comment:
Why is 33 picked here, maybe a comment or a constant declaration could help clarify this

Reply:
I think I copied that from the play-from-disk-h264 example. AFAIK this looks to be for maintain playback speed but since the webcam will be streaming live video we won't need it here. I'll get rid of it.